apps/server/package.json

```
@@ -7,7 +7,7 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "dev": "NODE_ENV=development pnpm --filter=@repo/database db:migrate:deploy && tsx watch --env-file=.env src/index.ts",
+    "cross-env": "NODE_ENV=development pnpm --filter=@repo/database db:migrate:deploy && tsx watch --env-file=.env src/index.ts",
```

```suggestion
    "dev": "cross-env NODE_ENV=development pnpm --filter=@repo/database db:migrate:deploy && tsx watch --env-file=.env src/index.ts",
```
I think this is the correct amendment, no? If you agree we also need to install cross-env in apps/server and add these changes on this PR